muckrock/agency/api_v2/viewsets.py

```diff
@@ -32,7 +32,7 @@ class Filter(django_filters.FilterSet):
         """API Filter for Agencies"""
 
         jurisdiction = django_filters.CharFilter(
```

Comment:
This should probably be a `ModelFilter` to properly specify that it should be a Jurisdiction ID.  There may have been some issues with `ModelFilter` - if that is still the case, this should at least be a `NumberFilter` and not a `CharFilter`.  We should try to get `ModelFilter` to work though for the best documentation.